### PR TITLE
Fix the build

### PR DESCRIPTION
**Pull request by rarguello**

### CODE REVIEW

**Comment by wolfc** [comment]
I don't have any build errors. What are you seeing?

**Comment by rarguello** [comment]
Some missing classes from Guava (com.google.common.*):
http://fpaste.org/0UwV/

After fixing that, you also need to add commons-cli.jar to the groovyc taskdef.

**Comment by wolfc** [comment]
Ah I see the issue I have checkstyle rpm installed.
This is actually very wicked. Bits in /etc/ant.d should not affect the classpath of the javac task.

File: bootstrap/build.xml [hunk 1]
```diff
@@ -169,6 +171,7 @@
             <pathelement path="/usr/share/java/jna.jar"/>
             <pathelement path="/usr/share/java/jnr-constants.jar"/>
             <pathelement path="/usr/share/java/jnr-posix.jar"/>
+            <pathelement path="/usr/share/java/guava.jar"/>
```
Comment (wolfc) [inline bootstrap/build.xml:174]: For future reference: please keep everything alphabetized to prevent unwarranted merge conflicts.

**Comment by wolfc** [comment]
Merged.

**Pull request closed without merging**